app/scenes/channel_info/channel_info.js

```diff
@@ -111,7 +158,7 @@ export default class ChannelInfo extends PureComponent {
                         <View style={[style.separator, {backgroundColor: this.props.theme.centerChannelBg}]}/>
                     </View>
                     <ChannelInfoRow
-                        action={() => true}
+                        action={() => this.handleLeave()}
```

Comment:
this row only be visible if the channel is other than `DEFAULT_CHANNEL` and not a DM